### DISCUSSION TITLE
关于RPC框架使用问题，欢迎提问

### DISCUSSION THREAD

**Discussion by ipipman** [ipipman/rpc-man#1] [Q&A] [answered]
关于RPC框架使用问题，欢迎提问

**Comment by chilumanxi** [comment]
项目有Maven仓库地址吗，我该怎么使用？

**Reply by ipipman** [reply, marked as answer]
请按照以下步骤操作：

1. 添加pom依赖，选择版本 rpcman.version：0.0.6

2. 在你的SpringBoot启动类配置 @Import({ProviderConfig.class, ConsumerConfig.class, RegistryCenterConfig.class}) 即可

3. 详细配置请参考README，注意：配置中心支持 Apollo、config-man， 注册中心支持 Zookeeper、registry-man，根据配置自主选择即可
config-man ： https://github.com/ipipman/config-man
registry-man ：https://github.com/ipipman/registry-man

**Reply by chilumanxi** [reply]
3ks

**Comment by ipipman** [comment]
项目已经传到中央仓库了，如果想直接使用的话，直接添加依赖即可，建议版本： rpcman.version：0.0.6

```java
<dependency>
    <groupId>io.github.ipipman</groupId>
    <artifactId>rpcman-core</artifactId>
    <version>${rpcman.version}</version>
</dependency>


```
